### ISSUE TITLE
signals station

### ISSUE THREAD

**Issue by amir-arad** [starwards/starwards#1208]
- [ ] has signals target state (in client only)

widgets:
 - [ ] long range radar ( blocked by #1204 )
 - [ ] target info (blocked by #1205 )
    - Lvl0: physics (distance, heading, rel.speed)
    - Lvl1: Faction, model
    - Lvl2: armor status, damage reports, list of systems
 - [ ] list of all signals jobs ( blocked by #1206 )

hotkeys: 
 - [ ] zoom in/out
 - [ ] targeting: 
     - next / prev target
     - toggle filters: unknown only (blocked by #1205 ), enemy only.
 - [ ] initiate scan on selected target (blocked by #1206 and #1205 )
 - [ ] initiate hack on selected target (per system) (blocked by #1206 and #1207 )
 - [ ] clear all jobs

**Comment by amir-arad** [comment]
Strategy update: Signals station is the priority new station for the first LARP event (4-station bridge: Pilot, Weapons, ECR, Signals). Navigator and Relay are deferred to post-first-event.

#1206 (signals jobs) is currently in progress. Once that lands, this station UI can be wired up.

See starwards-design/roadmap.md and starwards-design/decisions/004-ship-early-expand-later.md.

**Comment by amir-arad** [comment]
Backlog triage (2026-07-19): moved to the design KB — https://github.com/starwards/starwards-design/blob/main/product/backlog.md#signals--scanning

GitHub issues are now dispatch tickets only ([ADR-0003](https://github.com/starwards/starwards-design/blob/main/governance/decisions/0003-github-as-dispatch-only.md)). To pick this up again, open a fresh dispatch issue from the backlog entry (don't reopen this one).
